packages/path_provider/android/src/main/java/io/flutter/plugins/pathprovider/StorageDirectoryMapper.java

```diff
@@ -0,0 +1,51 @@
+package io.flutter.plugins.pathprovider;
+
+import android.os.Build.VERSION;
+import android.os.Build.VERSION_CODES;
+import android.os.Environment;
+
+/** Helps to map the Dart `StorageDirectory` enum to a Android system constant. */
+class StorageDirectoryMapper {
```

Comment:
I don't think we want this to be public.

Reply:
Without a specifier, this would be "package-private". Would you prefer if I move it into the plugin class?

Reply:
My bad it is good. For some reason I thought this is a dart file.